src/ray/gcs/gcs_server/gcs_resource_manager.cc

```diff
@@ -322,7 +322,10 @@ const absl::flat_hash_map<NodeID, std::shared_ptr<SchedulingResources>>
 
 void GcsResourceManager::SetAvailableResources(const NodeID &node_id,
                                                const ResourceSet &resources) {
-  cluster_scheduling_resources_[node_id]->SetAvailableResources(ResourceSet(resources));
+  auto iter = cluster_scheduling_resources_.find(node_id);
+  if (iter != cluster_scheduling_resources_.end()) {
+    iter->second->SetAvailableResources(ResourceSet(resources));
+  }
```

Comment:
Log a warning when `node_id` does not exist?

Reply:
Okey, I will add it soon.